softioc/softioc.py

```diff
@@ -240,10 +240,9 @@ def __call__(self):
 exit = Exiter()
 command_names.append('exit')
 
+# For backwards compatibility
+dbLoadDatabase = builder.dbLoadDatabase
```

Comment:
@Araneidae do you think I need to have this line in case anyone is calling `softioc.dbLoadDatabase`?

Reply:
Well, it is in the documented api, so yes, I think it would be better to leave it in.  After all, this is the method I was expecting you to be using...